source/views/components/filter/section-list.js

```diff
@@ -22,10 +22,10 @@ export function ListSection({filter, onChange}: PropsType) {
 	function buttonPushed(tappedValue: ListItemSpecType) {
 		let result
 
-		if (includes(selected, tappedValue)) {
+		if (some(selected, tappedValue)) {
```

Comment:
This should still be "includes".

Reply:
The reason I changed this to `some` is that `includes` compares objects by reference using `===`, which has worked so far with our filter objects because we haven't been creating new filter objects...However, while I was testing this branch I noticed that when the filters were updated from a recent filter combination, the filter view wouldn't display the selected filters correctly, even though everything on the Redux side of things was accurate. After some debugging, I narrowed it down to the uses of `includes` and `without` in this file, and changing them to `some` and `reject`, respectively, fixed the issue. Let me know if that doesn't make sense.

Reply:
I keep forgetting that these are objects, not primitives. Some is good in that case, but I'd like to use `selected.some(item => _.isDeepEqual(item, tappedValue))` instead of the matches shorthand, please.